tools/nl/embeddings/build_embeddings_v2_test.py

```diff
@@ -51,6 +51,27 @@ def _compare_files(t, output_path, expected_path):
       t.assertEqual(got, want)
 
 
+def _validate_sentence_to_dcid_map(t, df, dcid_col, sentence_columns):
+  sentence_dcid_map = {}
+  for _, row in df.iterrows():
+    dcid = row[dcid_col]
+
+    for col in sentence_columns:
+      for s in row[col].split(";"):
+        if not s:
+          continue
+        if s in sentence_dcid_map:
+          # Sentence already exists. It could either be a duplicate for the
+          # same SV dcid (which is Ok) OR the same sentence maps to a different
```

Comment:
Should we also catch duplicates for the same DCID (since they bloat up the index)?